### PR TITLE
Backend nondigital payment

### PR DESCRIPTION
**Pull request by ejohnst2**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -27,7 +27,7 @@ gem 'twitter'
 gem 'instagram'
 gem 'omniauth-google-oauth2'
 gem 'social-share-button'
-gem 'bootstrap-datepicker-rails'
+# gem 'bootstrap-datepicker-rails'
```
Comment (nicksarafa) [inline Gemfile]: lets just remove this

File: app/controllers/payments_controller.rb [hunk 2]
```diff
@@ -7,51 +7,64 @@ def new
...
-      # text hosts. phone numbers need to be added to database and repalced in to:
-      @client = Twilio::REST::Client.new ENV['TWILIO_ACCOUNT_SID'], ENV['TWILIO_AUTH_TOKEN']
-       @client.messages.create({
-         from: ENV['TWILIO_NUMBER'],
-         to: '+14254056694',
-         body: "New booking for #{order.trip.number_of_people} people starting #{order.trip.start_date}. Please login to your profile on http://www.troveko.org/users/sign_in and approve it."
-       })
-
-      order.update(payment: charge.to_json, state: 'paid')
+      order.update(state: 'paid')
```
Comment (nicksarafa) [inline app/controllers/payments_controller.rb]: `order -> state` previously used with `Stripe` gem

Comment (nicksarafa) [reply app/controllers/payments_controller.rb]: payments removed

File: app/controllers/payments_controller.rb [hunk 3]
```diff
@@ -7,51 +7,64 @@ def new
...
-     # creates customer in Skype based on token
-     customer = Stripe::Customer.create(
-      source: params[:token],
-      email:  current_user.email
-    )
-
-    # creates charge for order. does not charge credit card yet though (capture = false)
     @trip.orders.each do |order|
-
-      charge = Stripe::Charge.create(
```
Comment (nicksarafa) [inline app/controllers/payments_controller.rb]: remove `charge`

File: Gemfile [hunk 4]
```diff
@@ -40,7 +40,7 @@ gem 'postmark-rails'
 gem 'pundit'
 
 gem 'money-rails'
-gem 'stripe'
+# gem 'stripe'
```
Comment (nicksarafa) [inline Gemfile]: remove `stripe` gem

File: app/controllers/payments_controller.rb [hunk 5]
```diff
@@ -7,51 +7,64 @@ def new
...
+
   private
 
   def set_trip
     @trip = Trip.find(params[:trip_id])
   end
 
 end
+
+  # def create
```
Comment (nicksarafa) [inline app/controllers/payments_controller.rb]: 💣

File: app/models/order.rb [hunk 6]
```diff
@@ -5,11 +5,11 @@ class Order < ApplicationRecord
...
     order = Order.where(state: 'paid', id: id).find(self.id)
-    payment = JSON.parse(order.payment)
-    charge = Stripe::Charge.retrieve(payment["id"])
-    charge.capture
+    # payment = JSON.parse(order.payment)
+    # charge = Stripe::Charge.retrieve(payment["id"])
+    # charge.capture
 
-    order.update(payment: charge.to_json, state: 'fulfilled')
+    # order.update(payment: charge.to_json, state: 'fulfilled')
```
Comment (nicksarafa) [inline app/models/order.rb]: do we have to set the state to `fulfilled`

Comment (nicksarafa) [reply app/models/order.rb]: is `fulfilled` state being set matter to any other component in the app?

File: app/views/experiences/_list.html.erb [hunk 7]
```diff
@@ -9,7 +9,7 @@
 
     <div class="card-description">
       <h1><%= experience.title %></h1>
-      <p style="align;" id="price-card"><%= experience.duration %> days, <%= money_without_cents_and_with_symbol(experience.price) %> per person</p>
+      <p style="align;" id="price-card"><%= experience.duration %> days, Rp. <%= experience.price %> per person</p>
```
Comment (nicksarafa) [inline app/views/experiences/_list.html.erb]: `money-rails-gem` would come in handy when interpoltating moeny types

File: app/views/experiences/_map_box.html.erb [hunk 8]
```diff
@@ -4,7 +4,7 @@
     <div class="card_infobox" style="background-image: linear-gradient(rgba(0,0,0,0.3), rgba(0,0,0,0.2)), url('<%= cl_image_path experience.photos[0].path; %>');">
       <div class="card-description">
        <p><%= experience.city %></p>
-        <p><%= money_without_cents_and_with_symbol(experience.price) %> per night</p>
+        <p>Rp. <%= experience.price %> per night</p>
```
Comment (nicksarafa) [inline app/views/experiences/_map_box.html.erb]: need to display to `Rupiah` money type

File: app/views/experiences/_show_picture_carousel.html.erb [hunk 9]
```diff
@@ -20,6 +20,3 @@
       <span class="glyphicon glyphicon-chevron-right"></span>
     </a>
   </div>
-
- <%#= money_without_cents_and_with_symbol(@experience.fund.contributions.sum(:amount_cents) / 100) %>
- <%#=  money_without_cents_and_with_symbol(@experience.fund.funding_goal) %>
```
Comment (nicksarafa) [inline app/views/experiences/_show_picture_carousel.html.erb]: look into `oney_without_cents_and_with_symbol` if it works with Rupiah

**Comment by ejohnst2** [comment]
thumbs up

**Pull request merged**